README.md

```diff
@@ -1,89 +1,162 @@
-researchers should consider the numerical stability of the selected algorithm,
-and the platform on which the operation will be solved. The first analysis
-studies the accuracy of the solution while the second one compares the
-performances of the different implementations to select the best one.
-
-Nowadays, all the numerical computations are based on a set of standard
-libraries on which the most common operations are implemented. These libraries
-are different for dense matrices (BLAS, LAPACK, ScaLAPACK, ...) and for sparse
-matrices (SparseBLAS, ...). Moreover, there are  vendor implementations which
-are adjusted to the platform features:
-  - For multicores: ACML (AMD), ATLAS, Intel-MKL, OpenBLAS, ...
-  - For GPUs: cuBLAS (Nvidia), clBLAS, CLBlast, MAGMA, ...
-
-But, in any case, BLAS is always the lowest level in the hierarchy
-of numerical libraries, such that
+BLAS is always the lowest level in the hierarchy of numerical libraries, such that
 a good BLAS implementation improves the performances of all the other
-libraries.  The development of numerical libraries on SYCL is one of the most
-important objectives, because it will improve the performance of other SYCL
-applications. Obviously, it makes sense SYCL-BLAS was the first step in this
-task.
-
-On GPUs, the data communication to/from the device and the grain of the kernels
-play an important rule on the performances of the developments. On one
-hand, to reduce the communication cost, the most of the data should be mapped
-on the device, even the scalars. On the other hand, growing the size of the
-kernels allows the CPU to complete other tasks while the GPU is computing or to
-enter an energy-efficient C-state, reducing the energy consumption.
-
-To enlarge the grain of the kernels is a complex task, in which many aspects
-should be considered as the dependency between kernels, the grid topology, the
-grid sizes, etc. This complexity justifies that, usually, the fused kernels are
-manually written. An alternative to simplify this task could be to build a
-expression tree on which all the single operation which are required to solve a
-problem appears. This structure could be analysed by the compiler to decide how
-to merge the different kernel and the best grid topology to execute the fused
-kernel.  The use of expression trees is one of most important features of
-SYCL-BLAS.
-
-## Basic Concepts
+libraries.
+
+The SYCL-BLAS project is used by the [SYCL-DNN project](https://github.com/codeplaysoftware/SYCL-DNN) 
+project as a supporting library to perform some operations. SYCL-DNN is a library 
+implementing neural network operations.
+
+SYCL-BLAS has been developed in collaboration with the 
+*High Performance Computing & Architectures (HPCA) group*
+from the Universitat Jaume I [UJI](http://www.hpca.uji.es/).
+
+The project is maintained by [Codeplay Software](https://developer.codeplay.com).
+
+## Project Requirements
```

Comment:
I don't think this section reads very well. Should the 'project requirements' be an explicit list of dependent software (a sycl implementations, a reference blas library etc) or should it be a list of what is used to test the library (i.e. 'officially tested')? Currently it seems to be a mix of both.

How much of this is duplicated in the 'pre-requisites' below?

Reply:
I've moved things a bit, so there is a couple of sentences talking about these "requirements" in the introduction and then the Pre-requisites lists the mandatory pre-requisites.

Reply:
Building SYCL-BLAS as a library does not requires OpenBLAS at all. So the openBLAS is only there for verifying the result. So, I think to put it as a prerequisite for building a library can be confusing. Maybe we can explicitly mention in the pre-requisite that to verify a result you need to install a reference bals library like OpenBLAS or netlib BLAS, etc..

Reply:
I thought this is why my build failed the first time? I'll double check in that case.